salesforce/php/register.php

```diff
@@ -135,170 +15,60 @@ function add_permission_set($user_id){
 $email = strtolower($email);
 $email = trim($email);
 
-log_entry($email);
+
 $first = $_POST["first"];
 $last = $_POST["last"];
 
-$search_email = escapeEmail($email);
-$search_email = '{' . $search_email . '}';
-
-function create_interaction($first, $last, $email){
-    global $mySforceConnection;
-
-    $utm_source = '';
-    $utm_medium = '';
-    $utm_content = '';
-    $utm_campaign = '';
-
-    if( $_COOKIE['utm_source'] )
-        $utm_source = $_COOKIE['utm_source'];
-    if( $_COOKIE['utm_medium'] )
-        $utm_medium = $_COOKIE['utm_medium'];
-    if( $_COOKIE['utm_content'] )
-        $utm_content = $_COOKIE['utm_content'];
```

Comment:
is utm_content going away?

Reply:
PHP Storm marked it as not used, and I couldn't find where it was meant to be used so I got rid of it.

Reply:
PHP Storm marked it as not used, and I couldn't find where it was meant to be used so I got rid of it.